frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.html

```diff
@@ -10,6 +10,11 @@
     *ngIf="static"
     [textContent]="calendar.tooManyResultsText"
     class="op-wp-calendar--notification"></div>
+  <div
+    *ngIf="!hasData"
```

Comment:
I recommend you rebase this on dev and use `(principalIds$ | async).length` that way you don't have to concern yourself with the change detection mechanism.